galsim/bandpass.py

```diff
@@ -93,13 +94,13 @@ def __init__(self, throughput, blue_limit=None, red_limit=None, wave_type='nm',
         # `wave_list` is normally constructed (see `Bandpass.__mul__` below)
 
         self._orig_tp = throughput  # Save this for pickling.
-        self._tp = _tp              # This will normally become orig_tp turned into an actual 
-                                    # function (see _initialize_tp()), although in some cases,
-                                    # it can be supplied directly as a constructor argument.
-
```

Comment:
I'm not so attached to this one so I'll revert it (and any other similar cases).  I think the argument against is maintainability.  If you change the code or the comment, then it's (slightly) more work to realign than if the block comment occupied separate lines.